### DISCUSSION TITLE
'max_samples' parameter

### DISCUSSION THREAD

**Discussion by fcascao10** [zillow/quantile-forest#45] [Q&A] [answered]
It seems that the parameter 'max_samples' is not working correctly. Currently, it only permits to bootstrap the length of the total dataset. For instance, using a value other than 1.0 (for the float, e.g. 0.5) will raise the error: "ValueError: could not broadcast input array from shape ('# total obs * 0.5',) into shape ('# total obs',)".

**Comment by reidjohnson** [comment, marked as answer]
Thanks for identifying this issue! I've introduced updates that should address this in #47, so this should be fixed in the next release.
